### PR TITLE
Translated contributing.md

### PR DESCRIPTION
**Pull request by GarlicB**
I translated 'contributing.md' into english. If there are any awkward parts, please point out.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -1,65 +1,68 @@
...
 git fetch upstream
 git rebase upstream/master
-# 컨틀릭트 파일이 존재하여 리졸브가 필요했을 경우
+# if you need a resolve because of the conflict file,
 git rebase --continue
 ```
 
-## 이슈등록
-기존 코드에서 버그를 발견하셨거나 새로운 아이디어가 떠오르셨다면 먼저 이슈에 올려주시길 바랍니다. (이슈등록과 관련하여 자세한 내용은 [Issue Reporting](/docs/issue_reporting.md) 란을 참조) 등록한 뒤에는 패치를 준비하시기 이전에 먼저 코드오너와 함께 올리신 이슈에 대해서 이야기를 해보시길 권합니다. 코드오너가 누구인지는 [CODEOWNERS](https://github.com/lunchclass/absolute/blob/master/docs/CODEOWNERS)를 통해 확인하실 수 있습니다.
+## Creating_an_issue
```
Comment (nadongguri) [inline docs/contributing.md]: Is '_' required?

File: docs/contributing.md [hunk 2]
```diff
@@ -1,65 +1,68 @@
...
+If you've found a bug in existing code, or have come up with a new idea, please post it on the issue first.
+(For more information, see the [Issue Reporting](/docs/issue_reporting.md) section.)
+Talk to the code owners about your issue before patching them.
+You can find out who is the code owner through [CODEOWNERS](https://github.com/lunchclass/absolute/blob/master/docs/CODEOWNERS).
 
-## 패치준비
-### 패치사이즈
-패치는 가능하다면 적당히 작은 사이즈로 올려주시는 편이 좋습니다. 지나치게 많은 수정을 한 패치에 포함시키면 리뷰어가 좋은 코드리뷰를 하기가 어려워지며 리뷰과정에서 발견할 수 있는 이슈를 놓칠 확률이 높아집니다. 만약 새로운 구현으로 인해 수정사항이 많은 경우는 패치를 단계별로 작게 나누어서 올리시는 것도 좋은 방법입니다.
+## Patch
+### Patch_size
```
Comment (nadongguri) [inline docs/contributing.md]: Is '_' required?

Comment (GarlicB) [reply docs/contributing.md]: ok i'll fix it

**Comment by romandev** [comment]
@scarecrow21 Akshaj, could you please review this patch?
You seem an expert in English. :)
(or, after merging this patch as-is, you can send follow-up PR.)

**Comment by scarecrow21** [comment]
@romandev , can you guide me through reviewing this patch? I'm new to all this.

**Comment by romandev** [comment]
@scarecrow21 Thank you for reply.
It's very easy. You can just leave some inline comments in [Files Changed Tab](https://github.com/lunchclass/absolute/pull/483/files) in this patch if you think some point is wrong.

But I think it's better for you to make a new patch instead of reviewing this patch. WDYT?
(Because I welcome your contribution :) )

**Comment by romandev** [comment]
@scarecrow21, So, if you want a new patch, I'll merge this ASAP.

**Comment by scarecrow21** [comment]
Yep. Merge it. I'll take a new patch.

**Comment by scarecrow21** [comment]
So, I'll create a new branch, edit the docs/contributing.md file and submit a pull request?

**Comment by romandev** [comment]
@scarecrow21 Sure. Thanks!

**Comment by scarecrow21** [comment]
I have a doubt. The docs/contributing.md file has changed here on github but not on my local machine. What should I do so that the updated file is present on my local machine as well?

**Comment by romandev** [comment]
You can pull all changes as follows:
```
git pull
```

**Comment by romandev** [comment]
@scarecrow21 If you clone your own repo (scarecrow21/absolute instead of lunchclass/absolute), you can rebase as follows:

FYI, your own repo is ```scarecrow21/absolute``` and the repo is foked from ```lunchclass/absolute```.
Therefore upstream repo is ```lunchclass/absolute```.

You can set upstream repo as follows:
```
$ git remote add upstream https://github.com/lunchclass/absolute.git
```

Then, you can fetch upstream's changes into your .git internal storage as follows:
```
$ git fetch upstream
```

And then, you can rebase your repo based on upstream as follows:
```
$ git checkout master
$ git rebase upstream/master
```

And then, you can push your local repo to Github.
```
$ git push origin master -f
```

Then, you can see latest history in your Github repo.
Also, you can checkout a new branch to make a new patch.

If you have some question, please ping me anytime.
(Also, if you join our team's slack, I or others help you more.

Thank you very much!

**Comment by romandev** [comment]
FYI, this is Slack invitation link: https://bit.ly/lunchclass_slack

**Pull request merged**